src/transformers/configuration_gpt2.py

```diff
@@ -155,6 +157,9 @@ def __init__(
         self.summary_first_dropout = summary_first_dropout
         self.summary_proj_to_labels = summary_proj_to_labels
 
+        self.bos_token_id = bos_token_id
```

Comment:
cant u pass these to PretrainedConfig.__init__?

Reply:
oh maybe you deleted that stuff sorry if Im late to the party

Reply:
No you're right! The idea should be to put it into the config.json file instead of the configuration_<model>.py file